test/Microsoft.AspNet.BasicMiddleware.Tests/HttpMethodOverrideMiddleware.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.TestHost;
+using Xunit;
+
+namespace Microsoft.AspNet.BasicMiddleware.Tests
+{
+    public class HttpMethodOverrideMiddlewareTest
+    {
+        [Fact]
+        public async Task HttpMethodOverride_HeaderAvailable()
```

Comment:
Use consistent naming for this variable for all tests. Also, lowercase first word.